compiler/pipes/cfg.cpp

```diff
@@ -138,23 +154,39 @@ class CFG {
   }
 };
 
-Node CFG::new_node() {
-  Node res;
-  node_gen.init_id(&res);
+inline Node CFG::new_node() {
+  Node res(n_nodes++);
+
+  if ((n_nodes & 0x3f) == 1) {
```

Comment:
Can we make it a bit more understandable here? 
```suggestion
  if ((n_nodes & ((1 << 6) - 1)) == 1) {
```
Maybe it worth to extract constant with the mask value

Reply:
0x3f is really much understandable for everybody who knows hex notation, imho :)

Reply:
I think the option with `<<` highlights that it's the mask for power of two, rather than just some hex value
But OK, if you're not agree, just resolve this)

Reply:
more understandable is: `if (n_nodes % 64 == 1)`